server/dataset/cxg_dataset.py

```diff
@@ -23,7 +23,12 @@
 class CxgDataset(Dataset):
     # TODO:  The tiledb context parameters should be a configuration option
     tiledb_ctx = tiledb.Ctx(
-        {"sm.tile_cache_size": 8 * 1024 * 1024 * 1024, "sm.num_reader_threads": 32, "vfs.s3.region": "us-east-1"}
+        {
+            "sm.tile_cache_size": 16 * 1024 ** 3,
```

Comment:
will these changes be accompanied with a change of instance type?  The large tile_cache_size and py.init_buffer_bytes are suitable for a high-mem instance, but not for the current 64GB instances.  Thought: should these be pulled out into a deployment configuration (eg, set via env vars), or made adaptive (eg, tile cache is X% of RAM on the instance)?

Reply:
Yes there's an infra PR here: https://github.com/chanzuckerberg/single-cell-infra/pull/526
Re: making them dynamic based on instance - agreed!